test/test_source.py

```diff
@@ -654,3 +753,16 @@ def test_vuln_location():
     vl = VulnerabilityLocation.from_values("cpe:2.3:a:org.springframework:spring-web:*:*:*:*:*:*:*:*", "5.0.0.RC2",
                                            "*", "", "5.0.0.RC3")
     assert vl.version == ">=5.0.0.RC2-<5.0.0.RC3"
+
+
+def search_db(query):
```

Comment:
This looks like a useful method. Can this be moved to search.py?

Reply:
Yes, I believe so. I copied this from [cli.py](https://github.com/AppThreat/vulnerability-db/blob/master/vdb/cli.py#L179)